ee/vellum_ee/workflows/display/nodes/vellum/inline_prompt_node.py

```diff
@@ -72,7 +72,7 @@ def serialize(
                 "target_handle_id": str(self.get_target_handle_id()),
                 "variant": "INLINE",
                 "exec_config": {
-                    "parameters": raise_if_descriptor(node.parameters).dict(),
+                    "parameters": self._serialize_parameters(node.parameters, display_context),
```

Comment:
Remove this attribute from __unserializable_attributes__. Update any tests that regress bc of this change.